plugins/plugin-chart-pivot-table/README.md

```diff
@@ -0,0 +1,55 @@
+## @superset-ui/plugin-chart-pivot-table
+
+[![Version](https://img.shields.io/npm/v/@superset-ui/plugin-chart-pivot-table.svg?style=flat-square)](https://www.npmjs.com/package/@superset-ui/plugin-chart-pivot-table)
+
+This plugin provides Pivot Table for Superset.
+
+### Usage
+
+Configure `key`, which can be any `string`, and register the plugin. This `key` will be used to
+lookup this chart throughout the app.
+
+```js
+import PivotTableChartPlugin from '@superset-ui/plugin-chart-pivot-table';
+
+new PivotTableChartPlugin().configure({ key: 'pivot-table' }).register();
+```
+
+Then use it via `SuperChart`. See
+[storybook](https://apache-superset.github.io/superset-ui/?selectedKind=plugin-chart-pivot-table)
+for more details.
+
+```js
+<SuperChart
+  chartType="pivot-table"
```

Comment:
```suggestion
  chartType="pivot-table-v2"
```
There might be some other places to catch this as well, but let's give it a unique key so it can be loaded concurrently with the other one.

Reply:
Done